File: 268-missing-number/missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int sum1 = 0, sum2 = 0;
        for(auto n:nums){
            sum1+=n;
        }
        for(int i=1;i<=nums.size();i++){
            sum2+=i;
        }
        return sum2-sum1;
    }
};

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        sum1 = sum2 = 0
        n = len(nums)
        for i in range(n):
            sum1+=nums[i]
        sum2 = n*(n+1)//2
        return sum2-sum1